ppl/zqd/pcapanalyzer/process.go

```diff
@@ -28,27 +31,40 @@ func wrapError(err error, name, stderr string) error {
 	}
 	var pathErr *os.PathError
 	if errors.As(err, &pathErr) {
-		return fmt.Errorf("error executing %s runner: %s: %v", name, pathErr.Path, pathErr.Err)
+		return fmt.Errorf("error executing %s: %s: %v", name, pathErr.Path, pathErr.Err)
 	}
 	return err
 }
 
 type Process struct {
 	cmd       *exec.Cmd
 	stderrBuf *bytes.Buffer
+	stdoutBuf *bytes.Buffer
 }
 
 func NewProcess(cmd *exec.Cmd) *Process {
-	p := &Process{cmd: cmd, stderrBuf: bytes.NewBuffer(nil)}
-	// Capture stderr for error reporting.
+	p := &Process{cmd: cmd, stderrBuf: bytes.NewBuffer(nil), stdoutBuf: bytes.NewBuffer(nil)}
 	cmd.Stderr = p.stderrBuf
 	return p
 }
 
+func NewProcessStdout(cmd *exec.Cmd) *Process {
+	p := NewProcess(cmd)
+	p.cmd.Stdout = p.stdoutBuf
+	return p
+}
+
 func (p *Process) Start() error {
 	return wrapError(p.cmd.Start(), p.cmd.Path, p.stderrBuf.String())
 }
 
 func (p *Process) Wait() error {
 	return wrapError(p.cmd.Wait(), p.cmd.Path, p.stderrBuf.String())
 }
+
+func (p *Process) Stdout() string {
+	if p.stdoutBuf == nil {
```

Comment:
Given the current setup in `NewProcess`, will this ever be true?

Reply:
oops